#!/usr/bin/env python
PACKAGE = "project1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

euler = gen.const("Euler", int_t, 0, "Euler integration")
runge_kutta = gen.const("Runge_Kutta", int_t, 1, "Runge_Kutta integration")


int_enum = gen.enum([ euler, runge_kutta], "Enumeration of integration methods")
gen.add("int_method", int_t, 0, "Odometry integration method", 0, 0, 1, edit_method=int_enum)


exit(gen.generate(PACKAGE, "odometry", "parameters"))